z3c/recipe/usercrontab/__init__.py

```diff
@@ -18,6 +19,9 @@ def __init__(self, buildout, name, options):
         self.options = options
         options['entry'] = '%s\t%s' % (options['times'], options['command'])
         self.comment = options.get('comment')
+        if not get_bool(self.options, 'enabled', default=True):
+            self.options['entry'] = '# ' + self.options['entry']
```

Comment:
I have seen weird behaviour from buildout recipes, so I'll just throw it in here as a possible fix: don't do `self.options['entry']` here, but just `options['entry']`, so without the "self". It should be just the same, but I *have* seen difference crop up because of this. Shouldn't happen, but it might be worth looking at if a strange error persists.